tests/ipex/test_modeling.py

```diff
@@ -50,6 +50,7 @@
 
 SEED = 42
 torch.use_deterministic_algorithms(True)
+device_map = "xpu:0" if IS_XPU_AVAILABLE else None
```

Comment:
it would make sense to define this as "DEVICE", and then use it both for `device_map=DEVICE` and in the `.to(DEVICE)`
which should robustify the tests when xpu is available.

Reply:
Have updated the code per your suggestion.